apps/happy-blocks/block-library/education-header/style.scss

```diff
@@ -1,4 +1,4 @@
-@import "@automattic/calypso-color-schemes";
+@import "~@automattic/calypso-color-schemes";
```

Comment:
This actually reminds me of this previous PR I had: #52800. I'm curious why the esbuild sass plugin requires tilde imports

Reply:
We encountered an error where the esbuild sass plugin wasn't able to resolve the import and my guess is that it's due to sass not knowing about modules at all (I may be wrong) but it's some webpack `sass-loader` magic which does the resolving part (see https://webpack.js.org/loaders/sass-loader/).

Also, there's a big red banner on the sass documentation about discouring the usage of `@import` and recommending switching to `@use` (https://sass-lang.com/documentation/at-rules/import/).

Reply:
Ah yes! I was actually talking about that a couple years ago: p4TIVU-9Ho-p2 Since we never got forced to migrate to newer things, the work stalled out :) 

I was looking into switching more things to `@use`, and one of the blockers at the time was `@wordpress/base-styles` had some incompatible stuff (https://github.com/WordPress/gutenberg/issues/37044). I guess we need to revisit that at some point 😁 

But yeah, my impression at the time was that `~` was deprecated.